README.md

```diff
@@ -1,2 +1,6 @@
 # Backend Challenges Boilerplate - Basic Express
 [![Run on Repl.it](https://repl.it/badge/github/freeCodeCamp/boilerplate-express)](https://repl.it/github/freeCodeCamp/boilerplate-express)
+
+# When Developing on your Local Machine
+1) run `npm ci`
+2) `npm run start-local` (this preloads the environment variables set up in your `.env` file)
```

Comment:
1. We are moving all instructions out of the boilerplate and into the `/learn` environment.  This is to facilitate translation, since it's much easier to consolidate the text in one place.

    I could see adding a note to this challenge which explains that a package is required to pick up these values, as described in the original issue.

2. `npm install` is the preferred command for an initial install.
3. See my comments elsewhere about a local specific command.